rockylinux-9/Containerfile

```diff
@@ -0,0 +1,46 @@
+FROM rockylinux:9
+
+RUN dnf update -y \
+    && dnf install -y --allowerasing \
+      coreutils \
+      cpio \
+      dhclient \
+      e2fsprogs \
+      ethtool \
+      findutils \
+      initscripts \
+      ipmitool \
+      iproute \
+      kernel-core \
+      kernel-modules \
+      ncurses \
+      net-tools \
+      NetworkManager \
+      nfs-utils \
+      openssh-clients \
+      openssh-server \
+      pciutils \
+      policycoreutils-python-utils \
+      psmisc \
+      rsync \
+      rsyslog \
+      strace \
+      selinux-policy-targeted \
+      wget \
+      which \
+      words \
+      rdma-core \
+    && dnf clean all
+
+RUN touch /etc/sysconfig/disable-deprecation-warnings
```

Comment:
What warnings do you get without this? Are they things we could update in the image to avoid them by practice in stead?

Reply:
Good question. I kinda just copied the `rockylinux-8` example from this repo but never bothered to test or check if touching `/etc/sysconfig/disable-deprecation-warnings` does anything useful for rocky 9. I can try removing it in my test environment.

Reply:
Forgive me: I even checked the rockylinux-8 version for this and just didn't see it against other lines that weren't present. Now that I see this is just copied from there, I say you can just ignore this particular request: we can investigate this more holistically in the future across all versions.